src/redux/books/books.js

```diff
@@ -1,52 +1,73 @@
+import axios from 'axios';
+
```

Comment:
- [ ] `redux-thunk` should be added to the project as a package and used as middleware, kindly implement this
e.g.

     ```
      import { createStore, applyMiddleware } from 'redux';
      import thunk from 'redux-thunk';
      import rootReducer from './reducers/index';
      
          const store = createStore(
              rootReducer,
              applyMiddleware(thunk)
          );
     ```

Reply:
Hey @Ismailco!
I was trying to use the `createStore()` but in my IDEditor, I was receiving a warning like this method is deprecated we suggest you use `ConfigureStore()`. and there's a line through it. I mean the createStore. (~~createStore~~)

Reply:
@christianbiring1, I think you can use it with ConfigureStore() like this:

```
const store = configureStore({
  rootReducer,
  middleware: thunk
})
```